.changelog/4286.yml

```diff
@@ -0,0 +1,6 @@
+changes:
+- description: Removed lint step from all sdk build steps.
```

Comment:
this change doesnt seem outward facing to me to need a rn

Reply:
That's why it's tagged with 'type: internal'